roles/wordpress-install/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- include: directories.yml
+
+- name: Create .env file
+  template: src="env.j2"
+            dest="{{ www_root }}/{{ item.key }}/current/.env"
+            owner="{{ item.value.user }}"
+            group="{{ item.value.group }}"
```

Comment:
I'm pretty sure `item.value.user` and `item.value.group` will be undefined now that `user` and `group` are now removed from the `wordpress_sites` dict in `group_vars/development`.